### PR TITLE
Add cbrt

### PR DESCRIPTION
**Pull request by gwtnb**
Add `cbrt`.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: tests/cupy_tests/core_tests/test_fusion.py [hunk 1]
```diff
@@ -941,6 +942,9 @@ def test_sqrt(self):
         # numpy.sqrt is broken in numpy<1.11.2
         self.check_unary('sqrt')
 
+    def test_cbrt(self):
```
Comment (kmaehashi) [inline tests/cupy_tests/core_tests/test_fusion.py]: `@testing.with_requires('numpy>=1.10')` needed here.

**Comment by gwtnb** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**